data_structures/src/radon_error.rs

```diff
@@ -57,6 +57,16 @@ pub enum RadonErrors {
     ArrayIndexOutOfBounds = 0x70,
     /// Tried to access a value from a map using a key that does not exist
     MapKeyNotFound = 0x71,
+    // Bridge errors: errors that only belong in inter-client communication
```

Comment:
There is an error that has not been considered here. It was mentioned in the issue https://github.com/witnet/witnet-ethereum-bridge/issues/156:

> data request result length exceeds a bridge-contract-defined limit (in bytes) --> limits funds drainage

Regarding the error described by @lrubiorod about DRs with too many witnesses:

> Errors from calculate_weight -> Applying a limit of witnesses (100 to be safu) and BridgeTooManyWitnesses

I would include this type of error as a `BridgeMalformedRequest` if the calculated weight exceeds the block limit weight. Otherwise, I do not see why we should set a maximum at 100 and not 125 (theoretical maximum with 400 bytes DRs) or even lower.

Reply:
Instead of looking at the number of witnesses we can calculate the weight, this way the code will be more future proof if we change the block limit or the weight algorithm.